bindings/gumjs/gumdukinterceptor.c

```diff
@@ -221,6 +233,8 @@ _gum_duk_interceptor_dispose (GumDukInterceptor * self)
   _gumjs_duk_release_heapptr (self->core->ctx, self->invocation_context);
   _gumjs_duk_release_heapptr (self->core->ctx, self->invocation_args);
   _gumjs_duk_release_heapptr (self->core->ctx, self->invocation_retval);
+  _gumjs_duk_release_heapptr (self->core->ctx, self->cached_invocation_args);
+  _gumjs_duk_release_heapptr (self->core->ctx, self->cached_invocation_context);
```

Comment:
Let's follow the order of the struct, and add a blank line before these two so they're separated like in the struct.